docs/README.md

```diff
@@ -0,0 +1,36 @@
+
+# OJP Documentation

```

Comment:
Maybe use `# Generating OJP Documentation` instead to make it more clear from the header already this is not the documentation itself?